recipes/shellcheck/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -o xtrace -o pipefail -o errexit
+
+BINARY_HOME=${PREFIX}/bin
+PACKAGE_HOME=${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}
+
+mkdir -p ${BINARY_HOME}
+mkdir -p ${PACKAGE_HOME}
+
+export LIBRARY_PATH=${LIBRARY_PATH}:${PREFIX}/lib
```

Comment:
Why is this needed?

Reply:
Good point. It from https://github.com/conda-forge/git-annex-feedstock Need to clarify this.